editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx

```diff
@@ -21,6 +22,12 @@ import { DrawBoundary, PASSPORT_UPLOAD_KEY } from "../model";
 
 export type Props = PublicProps<DrawBoundary>;
 
+export type Boundary = GeometryObject | undefined;
+
+// Buffer applied to the address point to clip this map extent
+//   and applied to the site boundary and written to the passport to later clip the map extent in overview documents
+const BUFFER_IN_METERS = 75;
```

Comment:
I very much expect this number to change in user-testing, but it feels sufficiently conservative to start I think ?